docker-compose.yml

```diff
@@ -42,6 +57,8 @@ services:
 
   postgres:
     image: "postgres:9.4"
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
Yeah, nice one :+1: We should probably put these two lines  into effect regardless of your PR. It also broke the nav-container distribution when the DockerHub Postgres images changed their default auth method.